### ISSUE TITLE
Missing contentinfo landmark

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3573]
**Issue Description**: The footer links are in the main section.

**WCAG Criteria**: [Best Practice](https://www.w3.org/WAI/WCAG21/Understanding/)

**Notes**: Include the links that are about the page in a "contentInfo" landmark or in a <footer> HTML element. [MDN Guide on ContentInfo Role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/contentinfo_role).

<img width="1785" alt="Screenshot 2024-08-07 at 12 59 54 PM" src="https://github.com/user-attachments/assets/b7548b21-441e-429c-8f0a-9ad6dde50f19">

**Screenshot or Snippet**: [Link to Screenshot](https://share.zight.com/qGunPp96)

**Comment by danny-avila** [comment]
Closed by https://github.com/danny-avila/LibreChat/pull/3630
